### PR TITLE
Added Acrobat Reader DC Support

### PR DESCRIPTION
**Pull request by Tobias-B-Besemer**

### CODE REVIEW

File: cleaners/adobe_reader.xml [hunk 1]
```diff
@@ -27,8 +30,28 @@
     <label>Cache</label>
     <description>Delete the cache</description>
     <action command="delete" search="walk.files" path="~/.adobe/Acrobat/*/Cache"/>
-    <action command="delete" search="walk.files" path="$localappdata\Adobe\Acrobat\*\Cache"/>
-    <action command="delete" search="glob" path="$USERPROFILE\AppData\LocalLow\Adobe\Acrobat\9.0\Search\*.idx"/>
+    <action command="delete" search="walk.files" path="$LocalAppData\Adobe\Acrobat\*\Cache"/>
+    <action command="delete" search="glob" path="$UserProfile\AppData\LocalLow\Adobe\Acrobat\9.0\Search\*.idx"/>
+    <action command="delete" search="glob" path="$UserProfile\AppData\Local\Adobe\Acrobat\10.0\Cache\*"/>
```
Comment (az0) [inline cleaners/adobe_reader.xml]: This line and others look redundant to `$localappdata\Adobe\Acrobat\*\Cache` that was already there

File: cleaners/adobe_reader.xml [hunk 2]
```diff
@@ -41,10 +44,15 @@
...
     <action command="winreg" path="HKCU\Software\Adobe\Acrobat Reader\11.0\AVGeneral\cRecentFiles"/>
+    <action command="winreg" path="HKCU\Software\Adobe\Acrobat Reader\DC\AVGeneral\cRecentFiles"/>
+    <action command="winreg" path="HKCU\Software\Adobe\Acrobat Reader\DC\AVGeneral\cRecentFiles_old"/>
+    <action command="winreg" path="HKCU\Software\Adobe\Acrobat Reader\DC\AVGeneral\cRecentFolders"/>
   </option>
   <option id="tmp">
     <label>Temporary files</label>
     <description>Delete the temporary files</description>
     <action command="delete" search="glob" path="/dev/shm/sem.ADBE_*_$LOGNAME"/>
+    <action command="delete" search="glob" path="$localappdata\Temp\acrord32_sbx"/>
```
[the remark this answers is not in the document]
Comment (Tobias-B-Besemer) [reply cleaners/adobe_reader.xml]: It's a folder and it includes just files, so I thought 'glob' deletes all files in the folder? At least it works... :-D

File: cleaners/adobe_reader.xml (hunk 1 above)
Comment (Tobias-B-Besemer) [reply cleaners/adobe_reader.xml]: The version on my PC looks a bit different then the version you have mentioned... Is it the latest file?

But I see I can remove some lines if I work with paths like `$localappdata\Adobe\Acrobat\*\Cache`... I will do this and make a new push.

**Comment by Tobias-B-Besemer** [comment]
All checks have failed -> I make a new PR!

**Pull request closed without merging**